uint256.go

```diff
@@ -111,7 +189,7 @@ func (z *Int) Uint64() uint64 {
 
 // Uint64WithOverflow returns the lower 64-bits of z and bool whether overflow occurred
 func (z *Int) Uint64WithOverflow() (uint64, bool) {
-	return z[0], z[1] != 0 || z[2] != 0 || z[3] != 0
+	return z[0], (z[1] | z[2] | z[3]) != 0
```

Comment:
Either this isn't go-formatted, or the lines at 800 , 805 aren't go-formatted

Reply:
I double checked. It looks this is different depending if part of longer boolean expression (`&&`, `||`) or not.